src/store/actions/user.actions.js

```diff
@@ -32,10 +32,11 @@ export default {
   fetchMyProfileData: ({commit, state, getters}) => {
     let token = getters.getToken
     let config = {}
-    if (token) {
-      config.headers = {
-        'Authorization': 'Bearer ' + token
-      }
+    if (!token) {
```

Comment:
I know I haven't been a good job of this myself but I wonder if 1) we wan't to have an errors sections of the store we can set?

Reply:
I like that idea.